pages/common/sv.md

```diff
@@ -14,3 +14,11 @@
 - Get service status:
 
 `sudo sv status {{path/to/service}}`
+
+- Reload a service:
+
+`sudo sv reload {{path/to/service}}`
+
+- Start a serive only if it's not running and don't restart it if it stops
+
+`sudo sv once {{path/to/service}}` 
```

Comment:
This is an addition instead of fixing a `typo`.
Please create a separate PR which have a single logical change.